_posts/2018-06-29-14-04-netcore-7-0.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title : "14.4 OData Web API 7.0 now available"
+description: "7.0 release notes"
+category: "14. 7.x Features "
+---
+
+Today we're happy to announce the release of ASP.NET Web API OData 7.0 on the [NuGet gallery](https://www.nuget.org/)!
+
```

Comment:
Remove "Today" and just start with "We're happy to announce . . . ."

Reply:
Removed.